baybe/utils/dataframe.py

```diff
@@ -413,3 +414,42 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_printing_dataFrame(df: pd.DataFrame) -> str:
+    """Return a pretyt/readable str Object.
```

Comment:
typos and also having a bit my problem with capitalization

Reply:
Thank you! Typo fixed 👍